### DISCUSSION TITLE
Decameron transcription is launched !

### DISCUSSION THREAD

**Discussion by vicpsl** [PSL-Chartes-HTR-Students/TNAH-2021-DecameronFR#4] [General]
Decameron transcription is launched !

**Comment by sbiay** [comment]
Je me lance dans le zonage de mes folios !

**Comment by sbiay** [comment]
@vicpsl j'ai parcouru ta transcription et voici quelques notes 
[relecture_victor.odt](https://github.com/kristinkonstantinova/TNAH-2021-DecameronFR/files/7822390/relecture_victor.odt)
!

**Comment by vicpsl** [comment]
@sbiay Merci beaucoup, voici mon retour également.
Quelques tildes étaient déjà corrects, sans doute erreur de format dans odt ? Er certains ne sont pas très lisibles (semblent être décalés sur le caractère suivant). Sublime Text a un meilleur rendu.

Pour moi, reste surtout deux questions : être constant entre n(ot)re et u(ot)re et je lis bien "avoir" avec un "r" et non un "t" ligne 140. Le scribe a pu se tromper mais le HTR doit lire "r". Peut-on ajouter des corrections en notes ?

Plusieurs v/u avaient encore échappé à notre attention à tous les deux, j'ai effectué une recherche sur le caractère "v" pour être complet.
Bien à toi
[relecture_victor_VB.odt](https://github.com/kristinkonstantinova/TNAH-2021-DecameronFR/files/7833224/relecture_victor_VB.odt)

**Comment by sbiay** [comment]
Salut Victor,

Tu as raison pour *avoir* (sic). Nous ne sommes pas là pour éditer le texte. Laissons ainsi.

Qu'entends-tu par être constant ? Le scribe ne place pas toujours son tilde au même endroit…
